app/src/androidTest/kotlin/at/bitfire/davdroid/sync/SyncManagerTest.kt

```diff
@@ -512,6 +512,8 @@ class SyncManagerTest {
         collection: Collection = mockk<Collection>() {
             every { id } returns 1
             every { url } returns server.url("/")
+            every { pushSubscription } returns ""
```

Comment:
We don't need to return an empty string because it's not used when `pushSubscriptionExpires` is 0.

Also I'd rather return `null` because an empty string would mean: _there is a subscription URL, but it's empty_ (and thus invalid).